MekHQ/src/mekhq/gui/dialog/EditInjuryEntryDialog.java

```diff
@@ -328,16 +328,20 @@ private void setUserPreferences() {
     }
 
     private void btnOKActionPerformed(ActionEvent evt) {
-        injury.setTime(Integer.parseInt(txtDays.getText()));
-        injury.setHits(Integer.parseInt(txtHits.getText()));
-        injury.setFluff(txtFluff.getText());
-        injury.setLocation(((BodyLocationChoice) Objects.requireNonNull(ddLocation.getSelectedItem())).loc);
-        injury.setType(((InjuryTypeChoice) Objects.requireNonNull(ddType.getSelectedItem())).type);
-        injury.setPermanent(ddPermanent.getSelectedIndex() == 0);
-        injury.setWorkedOn(ddWorkedOn.getSelectedIndex() == 0);
-        injury.setExtended(ddExtended.getSelectedIndex() == 0);
-        injury.setUUID(UUID.randomUUID());
-        this.setVisible(false);
+        try {
+            injury.setTime(Integer.parseInt(txtDays.getText()));
+            injury.setHits(Integer.parseInt(txtHits.getText()));
+            injury.setFluff(txtFluff.getText());
+            injury.setLocation(((BodyLocationChoice) Objects.requireNonNull(ddLocation.getSelectedItem())).loc);
+            injury.setType(((InjuryTypeChoice) Objects.requireNonNull(ddType.getSelectedItem())).type);
+            injury.setPermanent(ddPermanent.getSelectedIndex() == 0);
+            injury.setWorkedOn(ddWorkedOn.getSelectedIndex() == 0);
+            injury.setExtended(ddExtended.getSelectedIndex() == 0);
+            injury.setUUID(UUID.randomUUID());
+            this.dispose();
+        } catch (Exception ex) {
+            logger.info("Failed to update injury entry", ex);
+        }
```

Comment:
Update to use the MathUtility methods for parseInt with default values where needed. That should take care of the exception portion.